sqlakeyset/serial/serial.py

```diff
@@ -82,13 +83,31 @@ def unescape(x):
     (datetime.time, "t"),
 ]
 
+
+class IDDict:
```

Comment:
Not super keen on this implementation - how about we just drop BUILTINS_INV entirely and go with a simple function?
```python
def invert_builtin(x) -> str:
    for k, v in BUILTINS:
        if x is v:
            return k
    raise ValueError(x)
```